fileuploader-kiwiirc-plugin/plugin.js

```diff
@@ -66,7 +67,21 @@ kiwi.plugin('fileuploader', function (kiwi, log) {
 
     const tokenManager = new TokenManager()
 
+    function loadLocale(_lang) {
+        try {
+            let lang = (_lang ? _lang : kiwi.i18n.language).split('-')
+            lang = lang[0] + '_' + lang[1].toUpperCase()
+            return require(/* webpackMode: "eager" */ '@uppy/locales/lib/' + lang)
```

Comment:
Can this be done with static import paths? Import paths that are determined at runtime can cause problems like webpack including unnecessary files in the bundle because it can't tell what's going to be needed, etc.